modules/new_model/shared/src/main/scala/observe/model/Conditions.scala

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package observe.model
+
+import cats.Eq
+import cats.derived.*
+import observe.model.enums.*
+
+final case class Conditions(
+  cc: CloudCover,
+  iq: ImageQuality,
```

Comment:
We have new classes (not enumerated) classes for conditions like `CloudExtinction`. Will we add those later on?

Reply:
Yes, we should do that.